view/frontend/web/js/model/step-navigator-mixin.js

```diff
@@ -0,0 +1,22 @@
+define([
+    'mage/utils/wrapper',
+    'jquery'
+], function (wrapper, $) {
+    'use strict';
+
+    let mixin = {
+        handleHash: function (originalFn) {
+            let hashString = window.location.hash.replace('#', '');
+
+            if (hashString.indexOf('venmo') >= 0) {
+                return false;
+            } else {
+                originalFn();
```

Comment:
Couldn't this work as a simpler way of doing the same thing?

Less JS is always better

```
var hashString = window.location.hash.replace('#', '');
return (hashstring.includes('venmo')) ? originalFn() : false;
```